libs/sdk-ui-dashboard/src/presentation/flexibleLayout/DashboardLayoutWidget.tsx

```diff
@@ -179,7 +179,7 @@ export const DashboardLayoutWidget: IDashboardLayoutWidgetRenderer<
     }
 
     function getGridColumnWidth(): number {
-        const columnWidthInGC = item.sizeForScreen(screen)?.gridWidth as number;
+        const columnWidthInGC = item.sizeForScreen(screen)?.gridWidth || 1;
```

Comment:
Can you create a constant into `libs/sdk-ui-dashboard/src/_staging/dashboard/flexibleLayout/config.ts`, named `DASHBOARD_LAYOUT_GRID_SINGLE_COLUMN` or some better name you can think of. Use it here and in the next file, please.